login.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet"  type="text/css" href="application.css">
+  </head>
+  <body>
+    <div class="background-login">
+      <div class="container-login">
+        <img class="login-logo" src="./assets/LogoWolox.png" alt="Wolox logo">
+        <div class="form-group">
+          <label for="mail">Email</label>
+          <input type="email" name="mail">
+          <label for="pass">Password</label>
+          <input type="password" name="pass">
+          <a href="list.html" class="btn">
+            <button class="btn-green">Login</button>
+          </a>
+        </div>
+        <div class="btn">
```

Comment:
If the contained button extends `btn` it's not necessary to apply this style. Maybe even this div isn't necessary :wink:

Reply:
Check @MMarussi comment, you shouldn't need this div at all